run/gem5art/run.py

```diff
@@ -65,6 +65,7 @@ class gem5Run:
     run_script_git_artifact: Artifact
     params: Tuple[str, ...]
     timeout: int
+    tags: str
```

Comment:
I expected this to be a list of strings (`List[str]`)

If it's only one tag, then the variable should be `tag`.

However, if we are only going to support one tag, I think we should use the `name` field instead.

Reply:
The difference between `tags` and name fields is that `name` is not modifiable, while `tags` is. (Now I think about it, it seems that we can modify the name.) The purpose of `tags` is like that of CONTAINER_NAME of docker: having a human readable way to recall a container/a collection of experiments.

I don't really know how MongoDB handles a list of strings, but I keep `tags` this way so that we can easily use regex to query the desired experiments.